samples/boards/nrf52/power_mgr/README.rst

```diff
@@ -0,0 +1,73 @@
+.. _nrf52-power-mgr-sample:
+
+Power management demo
+#####################
+
+Overview
+********
+
+This sample demonstrares a power manager app that uses the Zephyr
+power management infrastructure to enter into Low Power state.
+
+This app will cycle through following power schemes each time idle thread calls
+_sys_soc_suspend() hook function :-
+
+1. Low Power State:-
+   Low Power State is SOC specific state, In state is transparent to devices.
+   SOC and devices do not lose context in this Mode. SOC supports following
+   two Low Power states :-
+   A. CONSTANT LATENCY LOW POWER MODE : This Low Power State triggers CONSTLAT
+      task on nrf52 SOC. In this mode Low Exit Latency. In this Mode 16MHz Clock
+      is kept on.
+   B. LOW POWER STATE : In this Power State CONSTLAT task is triggered on nrf52
+      SOC. This is a deeper power state than CONSTANT LATENCY Mode. In this mode
+      16MHz Clock is turned off and only 32KHz clock is retained.
+
+2. Deep Sleep:- This Power State is mapped to System OFF state. In this mode
+   all devices on board gets suspended. All devices and SOC loses context on
+   wake up.
+
+3. No-op:- no operation, letting kernel idle.
+
+Requirements
+************
+
+This application uses nrf52 DK board for the demo.
+
+Building and Running
+********************
+
+.. code-block:: console
+
+    Building : make BOARD=nrf52_pca10040
```

Comment:
This would read better as three separate code blocks and putting the words that aren't command (Building, Flashing, and Running) outside the code block.  For example:

     Flashing (Use nrfjprog for flashing):

     .. code-block:: console

        nrfjprog --eraseall -f nrf52
        nrfjprog --program <zephyr.hex> -f nrf52
        nrfjprog --reset -f nrf52
 
     Running:

     .. code-block:: console